### ISSUE TITLE
IsItSnowing: Remove left padding

### ISSUE THREAD

**Issue by moollaza** [duckduckgo/zeroclickinfo-spice#1892]
This IA adds some left padding/margins causing it to be misaligned. This should be removed.

![isitsnowing](https://cloud.githubusercontent.com/assets/873785/8039035/1f9483ae-0dd5-11e5-85fd-d6085f4afc96.png)

Eventually this IA needs to be re-implemented like the IsItRaining IA: https://github.com/duckduckgo/zeroclickinfo-spice/pull/1739

**Comment by gokul1794** [comment]
If no one is working on this, I'd like to take it up!

**Comment by moollaza** [comment]
@gokul1794 no one has claimed it yet! Thanks a lot!

**Comment by gokul1794** [comment]
Umm, I'm not so sure as to where the IA is adding the left padding!
Will adding in the css file
>.zci__body{
padding-left=17px;}

Solve the problem?

**Comment by altern8tif** [comment]
Had a look at the css.  I think the issue stems from the padding on the `.zci.zci-snow.is-active` classes.  The right & left padding for the default text template is not zero, so the css shouldn't have values for that.

Suggested change:
````
.zci.zci--snow.is-active {
    margin-top: 0px;
    overflow: hidden;
    padding-top: 20px;
    padding-bottom: 20px;
}
````

Perhaps you could try to see if this works?

**Comment by gokul1794** [comment]
Yes, It does!

**Comment by moollaza** [comment]
Fixed in #1906
